app/views/articles/_full_comment_area.html.erb

```diff
@@ -28,11 +28,7 @@
 
         <div class="comments" id="comment-trees-container">
           <% if @article.comments_count > 0 %>
-            <% Comment.tree_for(@article, @comments_to_show_count).each do |comment, sub_comments| %>
-              <% cache ["comment_root_cached_tree", comment] do %>
-                <%= tree_for(comment, sub_comments, @article) %>
-              <% end %>
-            <% end %>
+            <%= render partial: "articles/comment_tree", collection: Comment.tree_for(@article, @comments_to_show_count), as: :comment_node, cached: proc { |comment, _sub_comments| comment } %>
```

Comment:
I had to provide a `proc` to cached to dictate how I want the cache key to be named. If I didn't provide it anything it has trouble properly providing a cache key.

Reply:
I usually specify them as stabby lambdas which I find a bit easier to read (`->(comment, _sub_comments) { comment }`) or does our Rubocop config not like that?

Reply:
@maestromac If we just return `comment` from the proc, does it end up making the cache key something like `Comment/123-#{updated_at}`?

Reply:
:smile: @jgaskins yup, as you said , it does and can been seen in the `QA Instructions` description.